<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Xoa_dau_sach</class>
 <widget class="QDialog" name="Xoa_dau_sach">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>629</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label_isbn">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>40</y>
     <width>39</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size: 15px;color:black;font-weight:700;
</string>
   </property>
   <property name="text">
    <string>ISBN </string>
   </property>
  </widget>
  <widget class="QLabel" name="label_tensach">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>40</y>
     <width>60</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size: 15px;color:black;font-weight:700;</string>
   </property>
   <property name="text">
    <string>Tên sách</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_tacgia_2">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>120</y>
     <width>191</width>
     <height>24</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>author</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget_danhmucsach_2">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>160</y>
     <width>491</width>
     <height>211</height>
    </rect>
   </property>
   <property name="editTriggers">
    <set>QAbstractItemView::EditTrigger::NoEditTriggers</set>
   </property>
   <attribute name="horizontalHeaderStretchLastSection">
    <bool>true</bool>
   </attribute>
   <column>
    <property name="text">
     <string>Mã sách</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Vị trí</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Trạng thái</string>
    </property>
   </column>
  </widget>
  <widget class="QLineEdit" name="lineEdit_ISBN_2">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>40</y>
     <width>131</width>
     <height>24</height>
    </rect>
   </property>
   <property name="maxLength">
    <number>17</number>
   </property>
   <property name="placeholderText">
    <string>ISBN-10 / ISBN-13</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_namsb_2">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>80</y>
     <width>81</width>
     <height>25</height>
    </rect>
   </property>
   <property name="maximum">
    <number>10000</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_theloai">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>80</y>
     <width>55</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size: 15px;color:black;font-weight:700;</string>
   </property>
   <property name="text">
    <string>Thể loại</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_sotrang_2">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>122</y>
     <width>81</width>
     <height>25</height>
    </rect>
   </property>
   <property name="maximum">
    <number>100000</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_sotrang">
   <property name="geometry">
    <rect>
     <x>42</x>
     <y>122</y>
     <width>59</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size: 15px;color:black;font-weight:700;</string>
   </property>
   <property name="text">
    <string>Số trang</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_theloai_2">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>80</y>
     <width>191</width>
     <height>24</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Science,realistic,...</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_tensach_2">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>40</y>
     <width>261</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>390</y>
     <width>168</width>
     <height>26</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QPushButton" name="pushButton_XoaDauSacg">
      <property name="text">
       <string>Xóa</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_cancel_2">
      <property name="text">
       <string>Cancel</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_namxb">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>95</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size: 15px;color:black;font-weight:700;</string>
   </property>
   <property name="text">
    <string>năm xuất bản</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_tacgia">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>120</y>
     <width>48</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size: 15px;color:black;font-weight:700;</string>
   </property>
   <property name="text">
    <string>Tác giả</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
